docs/source/usecases/WindowsHeader.md

```diff
@@ -0,0 +1,7 @@
+# Kokkos and `Windows.h`
+
+When using Kokkos on windows a program or library might include `windows.h`. This is problematic as this header defines two macros with the names `min` and `max` unless `NOMINMAX` is defined previously.
+The Preprocessor replaces strings in the source code with the macros yielding an uninterpretable result and thus compilation fails.
+Therefore, the header `Kokkos_Core.hpp` is protected against these macros, meaning they are undefined at the beginning and redefined at the end of the header file.
+Even though definitons inside `Kokkos_Core.hpp` are protected against the macros, code outside is not.
```

Comment:
Typo definitions

Reply:
done in [471940](https://github.com/kokkos/kokkos-core-wiki/pull/195/commits/4719403c528193e1de7cce5a49fd9106ac09aba5)